### ISSUE TITLE
Sonic Generations - Unknown STOP code: 0x0 (op=0x0, Out_MBox=empty)

### ISSUE THREAD

**Issue by Jonathan44062** [RPCS3/rpcs3#13072]
### Quick summary

Since no one has reported on this, then I will do the work, the game crashes with this error message 
```
F {SPU[0x4000100] Thread (RsdxPrimaryCellSpursKernel4) [0x065c0]} SIG: Thread terminated due to fatal error: Unknown STOP code: 0x0 (op=0x0, Out_MBox=empty)
(in file C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\rpcs3\Emu\Cell\SPUThread.cpp:5753[:22], in function stop_and_signal)
```
This happens when you play it for a certain time, making it unstable.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/10216230/RPCS3.log.gz)

### System configuration

Windows 10 21H2
Intel Core i7-8700
15.93 GiB RAM
GPU: NVIDIA GeForce GTX 1070 Ti (527.37)

**Comment by LukeWarnut** [comment]
Getting this in the demo about a minute in.

**Comment by Triticum0** [comment]
had the same issue when i tested the demo last week

**Comment by darkpro003** [comment]
hey guys does anyone know how to fix the crashing? i cant get passed the 2nd lvl... it keeps crashing no matter what setting i change.

**Comment by cas-1092** [comment]
> hey guys does anyone know how to fix the crashing? i cant get passed the 2nd lvl... it keeps crashing no matter what setting i change.

You cannot fix the crashing, Its the reason this github issue was made.

**Comment by darkpro003** [comment]
will it ever be fixed?

**Comment by Retrofreak90** [comment]
> Setting SPU Interpreter to either Dynamic or Static seems to prevent the crashing, but murders performance. Could anyone else test if that's the case? My log: [RPCS3.log.gz](https://github.com/user-attachments/files/20500737/RPCS3.log.gz)

Yep, I confirm this. I've played through two levels with no crash but, as you say, the speed is completely crippled.

**Comment by matheuswillder** [comment]
> Setting SPU Interpreter to either Dynamic or Static seems to prevent the crashing, but murders performance. Could anyone else test if that's the case? My log: [RPCS3.log.gz](https://github.com/user-attachments/files/20500737/RPCS3.log.gz)

Do you mean **CPU>SPU Decoder>** either **Interpreter (static)** or **Interpreter (dynamic)**?

If so, unfortunately, it keeps crashing randomly for me in the same way. I've tried several times, removing All Caches in each of them and even deleting the autosave that the game creates upon launch, but no dice.

In the brief moment the game started with **Interpreter (dynamic)**, it didn't seem that slow, but I don't know how it is as you progress through the level (it crashed right after).

Logs:

Interpreter (static):

[RPCS3.log.gz](https://github.com/user-attachments/files/23046071/RPCS3.log.gz)

Interpreter (dynamic):

[RPCS3.log.gz](https://github.com/user-attachments/files/23046073/RPCS3.log.gz)

**Comment by digant73** [comment]
@Jonathan44062 is that still an issue?

**Comment by alexconsole04** [comment]
Still an issue.
